src/components/Button/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { ReactComponent as PlusIcon } from 'assets/plus-icon.svg';
+import './Button.scss';
+
+const Button = ({ type, icon, children, link, onClick, disabled }) => {
+  let plusIcon = icon ? <PlusIcon className='button__plus-icon'/> : null;
```

Comment:
This isnt great, from my knowledge its bad practice to set `null` states on variables, can you rework this a bit

Reply:
It's set directly in the markup now to avoid using variables altogether.